src/libraries/System.Formats.Tar/tests/TarEntry/TarEntryPax.Tests.cs

```diff
@@ -36,6 +36,122 @@ public void Constructor_UnsupportedEntryTypes()
             Assert.Throws<InvalidOperationException>(() => new PaxTarEntry(TarEntryType.GlobalExtendedAttributes, InitialEntryName));
         }
 
+        [Fact]
+        public void Constructor_ConversionFromV7()
+        {
+            V7TarEntry v7 = new V7TarEntry(TarEntryType.V7RegularFile, InitialEntryName);
+            PaxTarEntry convertedV7 = new PaxTarEntry(other: v7);
+
+            Assert.Equal(TarEntryType.RegularFile, convertedV7.EntryType);
+            Assert.Equal(InitialEntryName, convertedV7.Name);
+        }
+
+
```

Comment:
Nit: It looks like about half of these tests have two blank lines between them instead of just one.